2021-06-28 18:48:45 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Table "DATABASECHANGELOGLOCK" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42102-197]
2021-06-29 06:12:03 jdbc[125]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:302)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:387)
	at clojure.java.jdbc$prepare_statement.invokeStatic(jdbc.clj:679)
	at clojure.java.jdbc$prepare_statement.invoke(jdbc.clj:626)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1112)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1093)
	at clojure.java.jdbc$query.invokeStatic(jdbc.clj:1182)
	at clojure.java.jdbc$query.invoke(jdbc.clj:1144)
	at toucan.db$query.invokeStatic(db.clj:288)
	at toucan.db$query.doInvoke(db.clj:284)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at toucan.db$simple_select.invokeStatic(db.clj:394)
	at toucan.db$simple_select.invoke(db.clj:383)
	at toucan.db$simple_select_one.invokeStatic(db.clj:420)
	at toucan.db$simple_select_one.invoke(db.clj:409)
	at toucan.db$select_one.invokeStatic(db.clj:627)
	at toucan.db$select_one.doInvoke(db.clj:620)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at toucan.db$select_one_field.invokeStatic(db.clj:636)
	at toucan.db$select_one_field.doInvoke(db.clj:629)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invokeStatic(cache.clj:96)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invoke(cache.clj:77)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invokeStatic(cache.clj:149)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invoke(cache.clj:126)
	at metabase.models.setting$db_or_cache_value.invokeStatic(setting.clj:185)
	at metabase.models.setting$db_or_cache_value.invoke(setting.clj:179)
	at metabase.models.setting$get_string.invokeStatic(setting.clj:198)
	at metabase.models.setting$get_string.invoke(setting.clj:188)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.util.i18n.impl$fn__14806$f__14807.invoke(impl.clj:169)
	at metabase.util.i18n.impl$site_locale_from_setting.invokeStatic(impl.clj:176)
	at metabase.util.i18n.impl$site_locale_from_setting.invoke(impl.clj:173)
	at metabase.util.i18n$site_locale.invokeStatic(i18n.clj:30)
	at metabase.util.i18n$site_locale.invoke(i18n.clj:27)
	at metabase.util.i18n$translate_site_locale.invokeStatic(i18n.clj:51)
	at metabase.util.i18n$translate_site_locale.doInvoke(i18n.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.util.i18n.SiteLocalizedString.toString(i18n.clj:74)
	at clojure.core$str.invokeStatic(core.clj:553)
	at clojure.core$str.invoke(core.clj:544)
	at metabase.core$destroy_BANG_.invokeStatic(core.clj:67)
	at metabase.core$destroy_BANG_.invoke(core.clj:64)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-06-29 06:12:04 jdbc[122]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.getAutoCommit(NewProxyConnection.java:1226)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.setAutoCommit(AttributeRestoringConnectionInvocationHandler.java:90)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:66)
	at com.sun.proxy.$Proxy28.setAutoCommit(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.getConnection(JobStoreSupport.java:788)
	at org.quartz.impl.jdbcjobstore.JobStoreTX.getNonManagedTXConnection(JobStoreTX.java:71)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3232)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-29 06:12:05 jdbc[125]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.C3P0ImplUtils.resetTxnState(C3P0ImplUtils.java:245)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.reset(NewPooledConnection.java:461)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.markClosedProxyConnection(NewPooledConnection.java:417)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.close(NewProxyConnection.java:87)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1111)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1093)
	at clojure.java.jdbc$query.invokeStatic(jdbc.clj:1182)
	at clojure.java.jdbc$query.invoke(jdbc.clj:1144)
	at toucan.db$query.invokeStatic(db.clj:288)
	at toucan.db$query.doInvoke(db.clj:284)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at toucan.db$simple_select.invokeStatic(db.clj:394)
	at toucan.db$simple_select.invoke(db.clj:383)
	at toucan.db$simple_select_one.invokeStatic(db.clj:420)
	at toucan.db$simple_select_one.invoke(db.clj:409)
	at toucan.db$select_one.invokeStatic(db.clj:627)
	at toucan.db$select_one.doInvoke(db.clj:620)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at toucan.db$select_one_field.invokeStatic(db.clj:636)
	at toucan.db$select_one_field.doInvoke(db.clj:629)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invokeStatic(cache.clj:96)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invoke(cache.clj:77)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invokeStatic(cache.clj:149)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invoke(cache.clj:126)
	at metabase.models.setting$db_or_cache_value.invokeStatic(setting.clj:185)
	at metabase.models.setting$db_or_cache_value.invoke(setting.clj:179)
	at metabase.models.setting$get_string.invokeStatic(setting.clj:198)
	at metabase.models.setting$get_string.invoke(setting.clj:188)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.util.i18n.impl$fn__14806$f__14807.invoke(impl.clj:169)
	at metabase.util.i18n.impl$site_locale_from_setting.invokeStatic(impl.clj:176)
	at metabase.util.i18n.impl$site_locale_from_setting.invoke(impl.clj:173)
	at metabase.util.i18n$site_locale.invokeStatic(i18n.clj:30)
	at metabase.util.i18n$site_locale.invoke(i18n.clj:27)
	at metabase.util.i18n$translate_site_locale.invokeStatic(i18n.clj:51)
	at metabase.util.i18n$translate_site_locale.doInvoke(i18n.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.util.i18n.SiteLocalizedString.toString(i18n.clj:74)
	at clojure.core$str.invokeStatic(core.clj:553)
	at clojure.core$str.invoke(core.clj:544)
	at metabase.core$destroy_BANG_.invokeStatic(core.clj:67)
	at metabase.core$destroy_BANG_.invoke(core.clj:64)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-06-29 06:12:06 jdbc[122]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:302)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:387)
	at jdk.internal.reflect.GeneratedMethodAccessor303.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at com.sun.proxy.$Proxy28.prepareStatement(Unknown Source)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectSchedulerStateRecords(StdJDBCDelegate.java:2947)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.findFailedInstances(JobStoreSupport.java:3294)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.clusterCheckIn(JobStoreSupport.java:3380)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3240)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-29 06:12:06 jdbc[122]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1512)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:516)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.rollback(NewProxyConnection.java:1033)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at com.sun.proxy.$Proxy28.rollback(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.rollbackConnection(JobStoreSupport.java:3639)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3264)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-29 06:12:07 jdbc[122]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.C3P0ImplUtils.resetTxnState(C3P0ImplUtils.java:245)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.reset(NewPooledConnection.java:461)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.markClosedProxyConnection(NewPooledConnection.java:417)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.close(NewProxyConnection.java:87)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.closeConnection(JobStoreSupport.java:3614)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.cleanupConnection(JobStoreSupport.java:3590)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3273)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-30 03:54:09 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:302)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:387)
	at clojure.java.jdbc$prepare_statement.invokeStatic(jdbc.clj:679)
	at clojure.java.jdbc$prepare_statement.invoke(jdbc.clj:626)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1112)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1093)
	at clojure.java.jdbc$query.invokeStatic(jdbc.clj:1182)
	at clojure.java.jdbc$query.invoke(jdbc.clj:1144)
	at toucan.db$query.invokeStatic(db.clj:288)
	at toucan.db$query.doInvoke(db.clj:284)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at toucan.db$simple_select.invokeStatic(db.clj:394)
	at toucan.db$simple_select.invoke(db.clj:383)
	at toucan.db$simple_select_one.invokeStatic(db.clj:420)
	at toucan.db$simple_select_one.invoke(db.clj:409)
	at toucan.db$select_one.invokeStatic(db.clj:627)
	at toucan.db$select_one.doInvoke(db.clj:620)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at toucan.db$select_one_field.invokeStatic(db.clj:636)
	at toucan.db$select_one_field.doInvoke(db.clj:629)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invokeStatic(cache.clj:96)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invoke(cache.clj:77)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invokeStatic(cache.clj:149)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invoke(cache.clj:126)
	at metabase.models.setting$db_or_cache_value.invokeStatic(setting.clj:185)
	at metabase.models.setting$db_or_cache_value.invoke(setting.clj:179)
	at metabase.models.setting$get_string.invokeStatic(setting.clj:198)
	at metabase.models.setting$get_string.invoke(setting.clj:188)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.util.i18n.impl$fn__14806$f__14807.invoke(impl.clj:169)
	at metabase.util.i18n.impl$site_locale_from_setting.invokeStatic(impl.clj:176)
	at metabase.util.i18n.impl$site_locale_from_setting.invoke(impl.clj:173)
	at metabase.util.i18n$site_locale.invokeStatic(i18n.clj:30)
	at metabase.util.i18n$site_locale.invoke(i18n.clj:27)
	at metabase.util.i18n$translate_site_locale.invokeStatic(i18n.clj:51)
	at metabase.util.i18n$translate_site_locale.doInvoke(i18n.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.util.i18n.SiteLocalizedString.toString(i18n.clj:74)
	at clojure.core$str.invokeStatic(core.clj:553)
	at clojure.core$str.invoke(core.clj:544)
	at metabase.core$destroy_BANG_.invokeStatic(core.clj:67)
	at metabase.core$destroy_BANG_.invoke(core.clj:64)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-06-30 03:54:10 jdbc[13]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.C3P0ImplUtils.resetTxnState(C3P0ImplUtils.java:245)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.reset(NewPooledConnection.java:461)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.markClosedProxyConnection(NewPooledConnection.java:417)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.close(NewProxyConnection.java:87)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1111)
	at clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1093)
	at clojure.java.jdbc$query.invokeStatic(jdbc.clj:1182)
	at clojure.java.jdbc$query.invoke(jdbc.clj:1144)
	at toucan.db$query.invokeStatic(db.clj:288)
	at toucan.db$query.doInvoke(db.clj:284)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at toucan.db$simple_select.invokeStatic(db.clj:394)
	at toucan.db$simple_select.invoke(db.clj:383)
	at toucan.db$simple_select_one.invokeStatic(db.clj:420)
	at toucan.db$simple_select_one.invoke(db.clj:409)
	at toucan.db$select_one.invokeStatic(db.clj:627)
	at toucan.db$select_one.doInvoke(db.clj:620)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at toucan.db$select_one_field.invokeStatic(db.clj:636)
	at toucan.db$select_one_field.doInvoke(db.clj:629)
	at clojure.lang.RestFn.invoke(RestFn.java:442)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invokeStatic(cache.clj:96)
	at metabase.models.setting.cache$cache_out_of_date_QMARK_.invoke(cache.clj:77)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invokeStatic(cache.clj:149)
	at metabase.models.setting.cache$restore_cache_if_needed_BANG_.invoke(cache.clj:126)
	at metabase.models.setting$db_or_cache_value.invokeStatic(setting.clj:185)
	at metabase.models.setting$db_or_cache_value.invoke(setting.clj:179)
	at metabase.models.setting$get_string.invokeStatic(setting.clj:198)
	at metabase.models.setting$get_string.invoke(setting.clj:188)
	at clojure.lang.Var.invoke(Var.java:384)
	at metabase.util.i18n.impl$fn__14806$f__14807.invoke(impl.clj:169)
	at metabase.util.i18n.impl$site_locale_from_setting.invokeStatic(impl.clj:176)
	at metabase.util.i18n.impl$site_locale_from_setting.invoke(impl.clj:173)
	at metabase.util.i18n$site_locale.invokeStatic(i18n.clj:30)
	at metabase.util.i18n$site_locale.invoke(i18n.clj:27)
	at metabase.util.i18n$translate_site_locale.invokeStatic(i18n.clj:51)
	at metabase.util.i18n$translate_site_locale.doInvoke(i18n.clj:48)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at clojure.lang.AFn.applyToHelper(AFn.java:154)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:660)
	at metabase.util.i18n.SiteLocalizedString.toString(i18n.clj:74)
	at clojure.core$str.invokeStatic(core.clj:553)
	at clojure.core$str.invoke(core.clj:544)
	at metabase.core$destroy_BANG_.invokeStatic(core.clj:67)
	at metabase.core$destroy_BANG_.invoke(core.clj:64)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-06-30 03:54:10 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.getAutoCommit(NewProxyConnection.java:1226)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.setAutoCommit(AttributeRestoringConnectionInvocationHandler.java:90)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:66)
	at com.sun.proxy.$Proxy28.setAutoCommit(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.getConnection(JobStoreSupport.java:788)
	at org.quartz.impl.jdbcjobstore.JobStoreTX.getNonManagedTXConnection(JobStoreTX.java:71)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3232)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-30 03:54:10 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:302)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:387)
	at jdk.internal.reflect.GeneratedMethodAccessor281.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at com.sun.proxy.$Proxy28.prepareStatement(Unknown Source)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectSchedulerStateRecords(StdJDBCDelegate.java:2947)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.findFailedInstances(JobStoreSupport.java:3294)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.clusterCheckIn(JobStoreSupport.java:3380)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3240)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-30 03:54:10 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1512)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:516)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.rollback(NewProxyConnection.java:1033)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at com.sun.proxy.$Proxy28.rollback(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.rollbackConnection(JobStoreSupport.java:3639)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3264)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
2021-06-30 03:54:10 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1526)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1502)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:476)
	at com.mchange.v2.c3p0.impl.C3P0ImplUtils.resetTxnState(C3P0ImplUtils.java:245)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.reset(NewPooledConnection.java:461)
	at com.mchange.v2.c3p0.impl.NewPooledConnection.markClosedProxyConnection(NewPooledConnection.java:417)
	at com.mchange.v2.c3p0.impl.NewProxyConnection.close(NewProxyConnection.java:87)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.closeConnection(JobStoreSupport.java:3614)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.cleanupConnection(JobStoreSupport.java:3590)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doCheckin(JobStoreSupport.java:3273)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.manage(JobStoreSupport.java:3857)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$ClusterManager.run(JobStoreSupport.java:3894)
